docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,282 @@
+as valid data during the phase unwrapping process. When phase unwrapping occurs over large expanses of water, it can
+lead to unexpected deformation signals or phase jumps in the unwrapped outputs, and the current masking approach
+does not correct for these impacts.
+
+#### Product Creation
+Image files are exported into the widely-used GeoTIFF format in a Universal Transverse Mercator (UTM) projection. Images
+are resampled to a pixel size that reflects the resolution of output image based on the requested number of looks: 80m
+pixel size for 20x4 looks, 40m for 10x2 looks, and 20m for 5x1 looks.
+
+Supporting metadata files are created, as well as a quick-look browse image.
+
+{% endblock %}
+
+{% block packaging %}
+## Product Packaging
+
+HyP3 Burst InSAR output is a zip file containing various files, including GeoTIFFs, a PNG browse image, a metadata file, and a README file.
+
+### Naming Convention
+
+The Burst InSAR product names are packed with information pertaining to the processing of the data, presented in the following order, as illustrated in Figure 4. 
+
+- The imaging platform name, always S1 for Sentinel-1.
+- Relative burst ID values assigned by ESA. Each value identifies a consistent burst footprint; relative burst ID values differ from one subswath to the next.
+- The acquisition dates of the reference (older) scene and the secondary (newer) scene
+- The polarizations for the pair, either HH or VV.
+- The product type (always INT for InSAR) and the pixel spacing, which will be either 80, 40, or 20, based upon the number of looks selected when the job was submitted for processing
+- The filename ends with the ASF product ID, a 4 digit hexadecimal number
+
+![Figure 4](../images/asf_burst_insar_names.png "Breakdown of ASF InSAR Naming Scheme")
+
+*Figure 4: Breakdown of ASF Burst InSAR naming scheme.*
+
+### Image Files
+# TODO: New Story Map
+All of the main InSAR product files are 32-bit floating-point single-band GeoTIFFs. To learn more about the rasters included in the product package, refer to the [Exploring Sentinel-1 InSAR StoryMap](https://storymaps.arcgis.com/stories/8be186e4125741518118d0102e6835e5 "Exploring Sentinel-1 InSAR StoryMap https://arcg.is/11DaW90" ){target=_blank} tutorial.
+
+- The *coherence* file pixel values range from 0.0 to 1.0, with 0.0 being completely non-coherent and 1.0 being perfectly coherent. 
+- The *unwrapped phase* file shows the results of the phase unwrapping process. Negative values indicate movement towards the sensor, and positive values indicate movement away from the sensor. This is the main interferogram output.
+- The *wrapped phase* file indicates the interferogram phase after applying the adaptive filter immediately before unwrapping. Values range from negative pi to positive pi.
+- The *look vectors* theta (θ) and phi (φ) describe the elevation and orientation angles of the sensor's look direction.
```

Comment:
@cirrusasf I think here under "Image Files" is the best place to describe the look vectors in more detail

Reply:
yes, I just edited "the look vectors" in Image Files section.